yui/dragdrop/dragdrop.js

```diff
@@ -16,7 +16,7 @@ YUI.add('moodle-qtype_ddmatch-dragdrop', function(Y, NAME) {
                 return;
             }
 
-            var containerid = 'div#q' + this.get('questionid'),
+            var containerid = 'div#' + this.get('questionid'),
```

Comment:
The new question div ids for 3.7 are compound of question-QuestionID-QuizPage

Reply:
Thank you for the update! (@nexterday )

Reply:
It’s still working in Moodle 3.9.  
Thanks a lot!

Reply:
You’re welcome.

Reply:
Hello, sorry but I am now retired and stopped all programming work. Feel free to fork this repository and make any change you want.